modules/mattermost-notification-workflowoperation/src/main/java/org/opencastproject/workflow/handler/mattermost/notification/MattermostNotificationWorkflowOperationHandler.java

```diff
@@ -128,6 +126,11 @@
    */
   public static final int SLEEP_SCALE_FACTOR = 2;
 
+  /**
+   * Gson instance for JSON serialization.
+   */
+  private static Gson gson = new GsonBuilder().create();
```

Comment:
Why not new Gson(); instead of new GsonBuilder().create(); ?

Reply:
I think it was my original recommendation since I usually use `GsonBuilder`. Since we do not make any configuration, you could use `new Gson()` in this case, but it really does not matter which you use, does it?

Reply:
It doesn't really matter, but Gson() would be shorter ;-)

I'm also fine with the longer variant.

Reply:
That's true :D